### ISSUE TITLE
Android (VR) version?

### ISSUE THREAD

**Issue by qwertychouskie** [Neverball/neverball#149]
Considering that GLES support already exists, it would be nice to see Neverball on Android.

Also, we live in the age of $10 Android-based VR headsets.  It would be really nice to see Neverball support these, as not everybody can afford multi-hundred dollar headsets like the Rift, but almost everybody has a smartphone and can afford $10.

**Comment by camthesaxman** [comment]
OpenHMD apparently supports Android, so theoretically, if someone manages to build a modern version of Neverball for Android and enables OpenHMD, it should support VR on Android.

**Comment by paolo-caroni** [comment]
There is unofficial neverball and neverputt port, but add android on official release would be beautiful...
Here you can see example of unofficial android repo:
https://github.com/drodin/neverball
https://github.com/gjtorikian/Neverball-ME

If there is an attempt for an official android release I can help with made different resolution of the logo, fastlane structure for fdroid market and maybe other stuff... but i'm not very good on coding.

**Comment by parasti** [comment]
Oh, wow! That first link is a nice find. Some guy seems to have ported Neverball and Neverputt to Android and is selling them on the Play Store for 89 eurocents. He also has .apk files for download on his website. And the source code is on Github. This happened in Febuary of this year. I had no idea about this. Thanks!

**Comment by paolo-caroni** [comment]
Yes, i have forked his repo today, i'm tryng to understand if his change brake other os release, he have made similar port of other open source game. I have his version of tux rider on my phone since 4 years ago and today I have installed neverball for try it.

**Comment by parasti** [comment]
I made a WebAssembly port of Neverball, which runs inside a modern desktop/mobile browser.

It is not 100% feature complete yet, but you can play a pre-release version here: https://neverball.github.io/

**Comment by armanelgtron** [comment]
Not useful now, but: I had a lower-end tablet around 2015 with an alternative app store, which contained a version of Neverball by "1orange". Was it just a build of Neverball-ME, or a different fork?

I tried looking it up, but all I can seem to find about it is slightly dodgy websites to download APKs that I don't trust, heh

**Comment by parasti** [comment]
The 1orange version really had nothing to do with us and was in violation of our license (GPL), which requires source code to be published.

**Comment by parasti** [comment]
> [@parasti](https://github.com/parasti) Is an official Android version planned? It would be nice to have if the next stable release of Neverball had an Android version.

Not really. https://play.neverball.org is the official mobile version, for now.
